src/python/pants/option/options.py

```diff
@@ -317,14 +317,25 @@ def for_scope(self, scope, inherit_from_enclosing_scope=True):
 
     return values
 
-  def get_fingerprintable_for_scope(self, scope):
+  def get_fingerprintable_for_scope(self, scope, include_passthru=False):
```

Comment:
I'm suggesting stop at ".. if the task ...". Since `Option.get_fingerprintable_for_scope` is public it may be used in non-Task non-Subsystem settings, we just don't know how its used. For one of these uses (here I agree its unlikely these exist), we're now changing what was fingerprinted in the past under your proposal. It could be their use case both has passthru args and fingerprinting them is exactly the wrong thing to do. I may be being paranoid, but it seems to me `:API: public` demands this.